src/options/subgraph.rs

```diff
@@ -43,6 +43,14 @@ pub struct OptionalSubgraphOpts {
     #[clap(long = "schema", short = 's')]
     #[serde(skip_serializing)]
     subgraph_schema_path: Option<Utf8PathBuf>,
+
+    /// The amount of time between introspection requests to the running subgraph.
+    ///
+    /// This argument is ignored if the `--schema` argument is passed.
+    /// The default value is 1 second.
+    #[clap(long = "polling-interval", short = 'i', default_value = "1")]
```

Comment:
```suggestion
    #[clap(long = "polling-interval", short = 'i', default_value = "1", conflicts_with = "schema")]
```

haven't tested this code, but clap has a [`conflicts_with`](https://docs.rs/clap/3.2.23/clap/struct.ArgGroup.html?search=conflict#method.conflicts_with) setting that we should be able to set here so that specifying both a polling interval and a schema results in a hard error

Reply:
that worked! I had to give a name to the schema option though :)